src/v1/language_service_client_config.json

```diff
@@ -2,51 +2,51 @@
   "interfaces": {
     "google.cloud.language.v1.LanguageService": {
       "retry_codes": {
+        "non_idempotent": [],
         "idempotent": [
           "DEADLINE_EXCEEDED",
           "UNAVAILABLE"
-        ],
-        "non_idempotent": []
+        ]
       },
       "retry_params": {
         "default": {
           "initial_retry_delay_millis": 100,
           "retry_delay_multiplier": 1.3,
           "max_retry_delay_millis": 60000,
           "initial_rpc_timeout_millis": 20000,
-          "rpc_timeout_multiplier": 1.0,
+          "rpc_timeout_multiplier": 1,
           "max_rpc_timeout_millis": 20000,
           "total_timeout_millis": 600000
         }
       },
       "methods": {
         "AnalyzeSentiment": {
-          "timeout_millis": 60000,
+          "timeout_millis": 600000,
```

Comment:
Can you double check that the gRPC service config actually says 600 seconds here?

Reply:
Yep - it's fine

Reply:
yes, we have 600,000 in grpc-config